.gitlab-ci.yml

```diff
@@ -37,7 +37,8 @@ noether-rocm:
     - echo '[{"subject":"/","metrics":[{"name":"Transfer Size (KB)","value":"19.5","desiredSize":"smaller"},{"name":"Speed Index","value":0,"desiredSize":"smaller"},{"name":"Total Score","value":92,"desiredSize":"larger"},{"name":"Requests","value":4,"desiredSize":"smaller"}]}]' > performance.json
 #    Note: PETSC_DIR is set by default in GitLab runner env, unsetting to isolate core tests
     - export PETSC_DIR= PETSC_ARCH=
-    - make -k -j$NPROC_CPU BACKENDS="$BACKENDS_CPU" junit realsearch=%
+    - make -k -j$NPROC_CPU BACKENDS="$BACKENDS_CPU /cpu/self/tmpl /cpu/self/tmpl/sub" junit search=t
+    - make -k -j$NPROC_CPU BACKENDS="$BACKENDS_CPU" junit search=ex
```

Comment:
What's the rationale for this manual split? I'm concerned that tests may get missed if any are introduced with a different naming convention.

Reply:
This is because the ex can't be run with the tmpl backends but the t should be.

I've come back around to the idea that we should perhaps just remove those backends. The delegation mechanism is already tested by the CPU family of backends.

Reply:
I'll merge this now to fix the CI and then save the question of removing the `/cpu/tmpl` backends (or leaving them as uncomplied examples? We have an example gallery QFunction, but I'm thinking we should perhaps remove it)